apps/common-app/src/apps/reanimated/examples/ExtrapolationExample.tsx

```diff
@@ -1,58 +1,40 @@
 import React from 'react';
 import { StyleSheet } from 'react-native';
-import type { PanGestureHandlerGestureEvent } from 'react-native-gesture-handler';
-import { PanGestureHandler } from 'react-native-gesture-handler';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
 import Animated, {
   Extrapolation,
   interpolate,
-  useAnimatedGestureHandler,
   useAnimatedStyle,
   useSharedValue,
   withTiming,
 } from 'react-native-reanimated';
 
 export default function ExtrapolationExample() {
-  const translation = {
-    x: useSharedValue(50),
-    y: useSharedValue(0),
-  };
-  type AnimatedGHContext = {
-    startX: number;
-    startY: number;
-  };
-  const gestureHandler = useAnimatedGestureHandler<
-    PanGestureHandlerGestureEvent,
-    AnimatedGHContext
-  >({
-    onStart: (_, ctx) => {
-      ctx.startX = translation.x.value;
-      ctx.startY = translation.y.value;
-    },
-    onActive: (event, ctx) => {
-      translation.x.value = ctx.startX + event.translationX;
-      translation.y.value = ctx.startY + event.translationY;
-    },
-    onEnd: (_) => {
-      translation.x.value = withTiming(50);
-      translation.y.value = withTiming(0);
-    },
-  });
+  const transY = useSharedValue(0);
```

Comment:
I wanted to avoid nitpicks but I see others don't thare this objection 😄 
```suggestion
  const translateY = useSharedValue(0);
```